### DISCUSSION TITLE
How to test an anonymous action in the form?

### DISCUSSION THREAD

**Discussion by luizmedeiros00** [filamentphp/filament#9241] [Help] [answered]
I have an anonymous action in the form and need to test if it's visible to the user.

![image](https://github.com/filamentphp/filament/assets/24945659/ca1216b9-cf2b-487b-94b6-c13dd5e8768f)
![image](https://github.com/filamentphp/filament/assets/24945659/2bc5fc12-556b-42bd-ba5a-617c850de61a)

**Comment by leandrocfe** [comment, marked as answer]
I think this assertion uses `mountAction` and an alternative would be using `mountFormComponentAction`
```php
->assertSee("mountFormComponentAction('attached_documento_oficial_com_fotoAction', 'attached_documento_oficial_com_foto')")
```


```

**Reply by luizmedeiros00** [reply]
It worked, thank you
